### PR TITLE
[Fix] Device Specific variable usage

### PR DESCRIPTION
**Pull request by tehAgitto**
# Pull Request summary:
Small refactor & fix

### CODE REVIEW

File: vmngclient/api/templates/models/cisco_system.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# from enum import Enum
```
Comment (sbasan) [inline vmngclient/api/templates/models/cisco_system.py:1]: Please remove or uncomment :)
Causes mypy `error: invalid syntax`

File: vmngclient/tests/templates/models/cisco_aaa.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# type: ignore
```
Comment (sbasan) [inline vmngclient/tests/templates/models/cisco_aaa.py:1]: Can we be more specific what type-check error we are ignoring?
Is it really needed for whole file?
(found also in other files)

**Review by sbasan — CHANGES_REQUESTED** [review]

**Comment by sbasan** [comment]
To fix mypy cache error modify linting.yml
```
mkdir .mypy_cache
...
poetry run mypy --show-error-codes --show-error-context --pretty --install-types --non-interactive vmngclient --cache-dir=.mypy_cache/
```

**Review by cicharka — APPROVED** [review]

**Review by sbasan — APPROVED** [review]
lgtm

**Pull request merged**